config/config.go

```diff
@@ -50,12 +50,23 @@ type Company struct {
 	People  []Account `json:"people" yaml:"people"`
 }
 
-// ClaSigners provides the overall structure of the CLA config: individual CLA
-// signers, bots, and corporate CLA signers.
-type ClaSigners struct {
+// ExternalClaSigners represents CLA signers managed by an external process,
+// i.e., not covered by this tool. This is useful for handling migrations into
+// or out of the system provided by Code Review Bot.
+type ExternalClaSigners struct {
```

Comment:
My rationale for having a separate External struct (rather than `External bool`) were as follows:

* we have a different bar for what we manage vs. external — namely, for self-managed CLAs, we have the user's full name + email in addition to their GitHub ID, while for external users, we can deal with a much smaller set of data, where the minimum bar is their GitHub ID (since the CLA is managed externally, everything else will be validated by that tool)

* if you're looking at the CLA config, it's obvious what's managed vs. external (since it's in a separate section)

* if you have a bool on a per-account basis, that means that Companies can have People, some of whom are managed locally, while some are managed externally, which seems strange: the company should move as a single atomic unit, since it's a single corporate CLA, and it should be managed atomically, rather than moving a person at-a-time